tests/integration/sentinel/transitions/sentinel-transition-error-log.spec.js

```diff
@@ -1,22 +1,20 @@
 const utils = require('@utils');
-const { expect } = require('chai');
 
 describe('Sentinel transition error log', async function() {
 
   after(async () => await utils.revertSettings(true));
 
-  it('log error for unkown transition', async () => {
+  it('log error for unknown transition', async () => {
     const unknownTransitionPattern = /Unknown transition "something"/;
     const settings = {
       transitions: {
         something: true
       }
     };
-    
-    const collectLogs = await utils.collectSentinelLogs(unknownTransitionPattern);
+
+    const collectLogs = await utils.waitForSentinelLogs(unknownTransitionPattern);
```

Comment:
Minor: name `collectLogs` could be updated, personally I'm fine with just `logs` something but more descriptive works too

Reply:
agree with the name change. I was lazy and just wanted to get the test fixed :D